tests/test_module_loader.py

```diff
@@ -45,7 +44,7 @@
 @pytest.mark.parametrize_any("editable_install", [True, False])
 def test_v2_module_loading(editable_install: bool, tmpdir: py.path.local, snippetcompiler, capsys, modules_v2_dir: str) -> None:
     # Work around caching problem in venv
-    feature_compiler_cache.set("False")
+    # feature_compiler_cache.set("False")
```

Comment:
This can probably go.
Before extending this test case I tried with it on and off and didn't see any difference.
Now that the test case actually checks that the caching takes place, we cannot set it to `False` anymore though.

Reply:
remove!

Reply:
I don't really remember. Let's remove it.